v4-client-js/src/clients/native.ts

```diff
@@ -1138,6 +1138,39 @@ export async function cctpWithdraw(squidPayload: string): Promise<String> {
   }
 }
 
+
+export async function cctpMultiMsgWithdraw(messages: { typeUrl: string; value: any }[]): Promise<string> {
```

Comment:
Looks like type error here. the Payload passed in will just be a `string`. and then you can do a `JSON.parse` similar to `cctpWithdraw`

Reply:
ah my bad, i see we call the `cctpWithdraw` input `squidPayload`